README.md

```diff
@@ -127,14 +127,8 @@ Run tests with all output from packages:
 lerna run test --stream
 ```
 
-Getting the Stitch integration tests running requires the 2 following env
-variables to be set:
-
-- `STITCH_TEST_APP_ID`
-- `STITCH_TEST_SERVICE_NAME`
-
-These can be gotten from the mongosh-test cluster in the Compass Atlas
-clusters.
```

Comment:
I have never seen or heard of these, and there is no mention of them in our source code. I assume this can be removed, but feel free to correct me if I’m wrong.